comms/examples/stress/service.rs

```diff
@@ -290,13 +301,48 @@ async fn start_initiator_protocol(mut conn: PeerConnection, protocol: StressProt
             framed.close().await?;
             println!("Done in {:.2?}. Closing substream.", start.elapsed());
         },
+        StressProtocolKind::BurstSend => {
+            let mut counter = 0u32;
+
+            println!(
+                "Sending {} messages in bursts of {} and finally {} messages ({} MiB)",
+                protocol.num_messages,
+                protocol.burst_size,
+                protocol.num_messages % protocol.burst_size,
+                protocol.num_messages * protocol.message_size / 1024 / 1024,
+            );
+            for _i in 0..(protocol.num_messages / protocol.burst_size) {
+                let mut iter = stream::iter(
```

Comment:
Given that there are no pauses between "bursts", is this not almost exactly the same as continuous send? A test I would like to see added is concurrent reading and writing.

Reply:
> A test I would like to see added is concurrent reading and writing.

This is easily simulated by starting a longish test from both ends within a short time span; no additional code changes needed.

Reply:
> Given that there are no pauses between "bursts", is this not almost exactly the same as continuous send?

I guess the only difference will be finalization given by `send_all` after each burst, i.e. `A future that completes after the given stream has been fully processed into the sink, including flushing.`